_studio/shared/umc/codec/jpeg_common/src/bitstreamin.cpp

```diff
@@ -113,7 +106,7 @@ JERRCODE CBitStreamInput::FillBuffer(int nMinBytes)
     m_currPos = 0;
   }
 
-  if(!m_eod)
+  if(!m_eod && nullptr != m_in.get())
```

Comment:
shared_ptr / unique_ptr have operator bool (), which you can use to check if it isn't empty:
```
if(!m_eod && m_in)
```